src/decompress.rs

```diff
@@ -610,6 +618,20 @@ impl CompressedBlock {
         mut output_index: usize,
         queued_output: &mut Option<QueuedOutput>,
     ) -> Result<(CompressedBlockStatus, usize), DecompressionError> {
+        // `litlen_table_mask` (and `dist_table_mask`) calculation assumes that `LITLEN_TABLE_SIZE`
+        // (or `DIST_TABLE_SIZE`) is a power of two.
+        assert!(LITLEN_TABLE_SIZE.count_ones() == 1);
```

Comment:
Doesn't make a huge difference, but there's actually a [`is_power_of_two`](https://doc.rust-lang.org/std/primitive.u32.html#method.is_power_of_two) method.

Reply:
and this could be in `const {}` to check at compile time